tools/haskell_module_worker/README.md

```diff
@@ -0,0 +1,60 @@
+This is a work-in-progress implementation of a persistent worker for
+the `haskell_module` rule.
+
+To run `haskell_module` tests with it run
+```
+USE_BAZEL_VERSION=6.0.0-pre.20220520.1 bazelisk test \
+  --experimental_worker_multiplex_sandboxing \
+  --worker_max_instances=1 \
+  --worker_quit_after_build \
+  --define use_haskell_module_worker=True \
+  --worker_verbose \
+  --experimental_worker_max_multiplex_instances=2 \
+  --worker_extra_flag --num-workers=2 \
+  //tests/haskell_module/dep-narrowing-th/..
+```
+
+Bazel 6 is necessary to use the flag `--experimental_worker_multiplex_sandboxing`.
```

Comment:
I'm guessing adding tests will be difficult because of this restriction?

Reply:
Yeah, or run without a sandbox, or run it in singleplex mode instead.